ios/CalendarMobile/Info.plist

```diff
@@ -1,55 +1,59 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
+<dict>
```

Comment:
please explain this dict change

Reply:
Auto refactoring done by Prettier while appending UIAppFonts keys.